service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/workspace/LaunchCloneControlledGcpBigQueryDatasetResourceFlightStep.java

```diff
@@ -61,6 +62,11 @@ public StepResult doStep(FlightContext context) throws InterruptedException, Ret
         ControlledResourceKeys.CLONING_INSTRUCTIONS, resource.getCloningInstructions());
     subflightInputParameters.put(ResourceKeys.RESOURCE, resource);
     subflightInputParameters.put(ControlledResourceKeys.LOCATION, location);
+    subflightInputParameters.put(WorkspaceFlightMapKeys.WORKSPACE_ID, destinationWorkspaceId);
+    subflightInputParameters.put(
+        JobMapKeys.DESCRIPTION.getKeyName(),
+        String.format("Clone BigQuery Dataset %s", resource.getResourceId().toString()));
```

Comment:
Would it be helpful for this (and other clone resource flight) description to also include the destination workspace ID? maybe a question for @cbookg

Reply:
I figured the destination workspace ID is how they'll be found to begin with. I'm also wondering if I shouldn't have a more regular format for the descriptions that just plain English, though I believe that's what Chris is parsing today.